;

```diff
@@ -0,0 +1,386 @@
+# The MIT License (MIT)
```

Comment:
@dherrada What is this file? Did you accidentally rename something or did you duplicate a file?

Reply:
looks like it was a duplicate of adafruit_fram.py. I've removed it.